pkg/kubelet/sysctl/whitelist.go

```diff
@@ -22,14 +22,10 @@ import (
 
 	"k8s.io/kubernetes/pkg/apis/core/validation"
 	policyvalidation "k8s.io/kubernetes/pkg/apis/policy/validation"
+	"k8s.io/kubernetes/pkg/kubelet/events"
```

Comment:
For anyone else reading this... `pkg/kubelet/events` has no dependencies (is really just a list of constants). So there aren't any concerns about requiring it :)